src/imagery/i.landsat/i.landsat.download/i.landsat.download.py

```diff
@@ -204,110 +188,174 @@ def main():
         else:
             gs.fatal(_("Output directory <{}> is not a directory").format(outdir))
     else:
-        outdir = "/tmp"
+        outdir = os.getcwd()
 
     # Download by ID
     if options["id"]:
-
-        ids = options["id"].split(",")
-
-        ee = EarthExplorer(user, password)
-
-        for i in ids:
-
-            try:
-
-                ee.download(
-                    identifier=i, output_dir=outdir, timeout=int(options["timeout"])
+        # Should use other provider other than USGS,
+        # as there was a bug in USGS API, fixed here
+        # https://github.com/CS-SI/eodag/issues/1252
+        # TODO: set provider to USGS when the above changes goes into production
+        if options["datasource"] == "usgs" and not (
+            eodag.__version__ == "3.0.0b3" or eodag.__version__ >= "3.0.0"
```

Comment:
You should check versions as numbers or tuples of numbers, assuming semantic versioning. I do not think >= works with strings. Also no need to check for specific beta versions...